config/currency_iso.json

```diff
@@ -1155,8 +1155,8 @@
     "priority": 100,
     "iso_code": "KWD",
     "name": "Kuwaiti Dinar",
-    "symbol": "د.ك",
-    "alternate_symbols": ["K.D."],
+    "symbol": "K.D. ",
```

Comment:
Didn't mean to create a pull request, it's closed.

On 24 March 2016 at 13:53, printercu <notifications@github.com> wrote:

> In config/currency_iso.json
> <https://github.com/RubyMoney/money/pull/617#discussion_r57318718>:
>
> > @@ -1155,8 +1155,8 @@
> >      "priority": 100,
> >      "iso_code": "KWD",
> >      "name": "Kuwaiti Dinar",
> > -    "symbol": "د.ك",
> > -    "alternate_symbols": ["K.D."],
> > +    "symbol": "K.D. ",
>
> Here is space after the last dot. Is it ok?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/RubyMoney/money/pull/617/files/2afea67900324e01145d4caa22dd34cfe0050d3e#r57318718>
>



-- 
Mark Jacobs
*Trainee Developer*


<http://www.current-rms.com/> <http://www.current-rms.com/>
<https://www.facebook.com/currentrms/> <https://twitter.com/currentrms>
<https://www.linkedin.com/company/current-rms>

<http://www.current-rms.com/shows/>

*An inspHire Ltd <http://www.insphire.com/> product*